app/build.gradle

```diff
@@ -92,6 +92,7 @@ android {
 
 	lintOptions {
 		checkReleaseBuilds false
+		abortOnError false
```

Comment:
Is this still needed? I think better to disable the lint checks that fail for now until we can resolve them.

Reply:
Yeah, this is probably better. I've addressed the lint errors in the force-pushed version of the branch.